src/compiler/comments.ts

```diff
@@ -198,9 +198,9 @@ namespace ts {
             }
 
             // Leading comments are emitted at /*leading comment1 */space/*leading comment*/space
-            emitPos(commentPos);
+            if (emitPos) emitPos(commentPos);

```

Comment:
why this check is necessary? If `emitPos` can be undefined, can you please reflect this in its type?